spec/requests/compliments_spec.rb

```diff
@@ -13,4 +13,17 @@
       }
     }.as_json)
   end
+
+  it 'returns a random compliment if the id is random' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.